ui/app/src/services/contentTypes.ts

```diff
@@ -40,15 +40,17 @@ const typeMap = {
   'image-picker': 'image'
 };
 
-const systemValidationsNames = ['itemManager', 'minSize', 'maxSize', 'maxlength', 'readonly'];
+const systemValidationsNames = ['itemManager', 'minSize', 'maxSize', 'maxlength', 'readonly', 'width', 'height'];
 
 const systemValidationsKeysMap = {
   minSize: 'minCount',
   maxSize: 'maxCount',
   maxlength: 'maxLength',
   contentTypes: 'allowedContentTypes',
   tags: 'allowedContentTypeTags',
-  readonly: 'readOnly'
+  readonly: 'readOnly',
+  width: 'width',
+  height: 'height'
```

Comment:
Do we currently have min/max width/height? Do we have both exact width/height & max/min or just one?

Reply:
Added new fields for minWidth, maxWidth, minHeight and maxHeight